### PR TITLE
CI-86c0jz8p7_basic_info_validation

### PR DESCRIPTION
**Pull request by marvinIsSacul**

### CODE REVIEW

File: packages/users-app/src/screens/create.user.component.tsx [hunk 1]
```diff
@@ -90,14 +67,28 @@ function AddUserRole() {
...
         <Typography
           variant="subtitle1"
           fontSize={16}
           gutterBottom
           fontWeight={400}
+          sx={{
+            fontSmooth: 'never',
+            '-webkit-font-smoothing': 'antialiased',
+            '-moz-osx-font-smoothing': 'grayscale'
+          }}
```
Comment (drizzentic) [inline packages/users-app/src/screens/create.user.component.tsx:91]: We need to adapt this to the new Base template

File: packages/users-app/src/screens/list.users.screen.tsx [hunk 2]
```diff
@@ -142,13 +142,19 @@ function UsersList() {
...
         Manager Users
       </Typography>
 
       <Grid container>
         <Grid item xs={11}>
-          <Typography variant="subtitle1" gutterBottom>
+          <Typography
+            variant="subtitle1"
+            gutterBottom
+          >
```
Comment (drizzentic) [inline packages/users-app/src/screens/list.users.screen.tsx:157]: Adapt to new basetemplate

**Review by drizzentic — CHANGES_REQUESTED** [review]

**Comment by marvinIsSacul** [comment]
Task linked: [https://app.clickup.com/t/86c0j3aan](https://app.clickup.com/t/86c0j3aan)

File: packages/users-app/src/components/common/basic.info.component.tsx [hunk 3]
```diff
@@ -162,23 +257,28 @@ export function BasicInfo(props: {
...
+                        checked={user.settings?.list?.autoupdate}
+                        onChange={e => {
+                          const newUser = structuredClone(user)
+                          newUser.settings.list.autoupdate = e.target.checked
+                          setUser(newUser)
+                        }}
                       />
                     }
-                    label="Auto-update Transaction List."
+                    label="Auto-update Transaction LisshowTooltipst."
```
[the remark this answers is not in the document]
Comment (marvinIsSacul) [reply packages/users-app/src/components/common/basic.info.component.tsx]: Yea. Fixed it now.

**Review by drizzentic — APPROVED** [review]
LGTM!

**Pull request merged**